brave/browser-laptop
# Refactor context menu

Closes #11354

Auditors: @cezaraugusto

Test Plan:
1. `npm run test -- --grep='ContextMenu'`
2. `npm run test -- --grep='bookmarksToolbar'`
3. `npm run test -- --grep='tab tests'`

For QA team:
- Make sure context menu items in the URL bar work
- Make sure context menu items on content work with no selected text.
- Make sure context menu items on content work with selected text.
- Make sure context menu items on content work inside an editable control on about:styles (input, textarea, or contenteditable).

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
